src/codecs.ts

```diff
@@ -119,6 +121,14 @@ export const DataCategoryInput = t.type({
   name: t.string,
 });
 
+export const Attributes = t.type({
+  key: t.string,
+  values: t.array(t.string),
```

Comment:
@michaelfarrell76 I noticed the upsert accepts the values as a list, so I just updated the values field into an array of string